geonode_mapstore_client/client/js/components/ActionNavbar/ActionNavbar.jsx

```diff
@@ -80,44 +80,28 @@ const ActionNavbar = forwardRef(
             variant,
             size,
             resource,
-            titleItems
+            titleItems,
+            disableTitle
         },
         ref
     ) => {
         return (
             <nav ref={ref} className={`gn-menu gn-${variant}`} style={style}>
                 <div className="gn-menu-container">
                     <div className="gn-menu-content">
-                        <div className="gn-action-navbar-title">
-                            <div>
-                                <NavLink
-                                    href="#"
-                                    className="gn-action-navbar-breadcrumb-link"
-                                >
-                                    <FaIcon name="home" />
-                                </NavLink>
-                                <FaIcon
-                                    name="angle-right"
-                                    className="gn-action-navbar-breadcrumb-seperator"
-                                />
-                                <p
-                                    title={resource?.title}
-                                    className="gn-action-navbar-resource-title"
-                                >
-                                    {resource?.title}
-                                </p>
-                            </div>
-                            {titleItems.map(({ Component, name }) => (
-                                <Component key={name} variant="info"/>
-                            ))}
-                        </div>
+                        <BreadCrumb
+                            resource={resource}
+                            titleItems={titleItems}
+                            disableTitle={disableTitle}
```

Comment:
This is not a request of change but I was thinking if it was better pass the `disableTitle` or not because we could solve this also with
```js
disableTitle ? null : <BreadCrumb
    resource={resource}
    titleItems={titleItems}
/>
```